docs/gen-ai/llm-spans.md

```diff
@@ -65,26 +82,380 @@ These attributes track input data and metadata for a request to an LLM. Each att
+| [`gen_ai.event.content`](/docs/attributes-registry/gen-ai.md) | string | The GenAI event payload serialized into JSON string. | `[{'content': 'What is the capital of France?'}]`; `plain string` | `Conditionally Required` [1] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 
-**[1]:** It's RECOMMENDED to format completions as JSON string matching [OpenAI messages format](https://platform.openai.com/docs/guides/text-generation)
+**[1]:** If and only if recording GenAI events as span events. It is NOT RECOMMENDED to record GenAI events with span event API and SHOULD only be used if Event API is not yet available in the corresponding OpenTelemetry API package.
 <!-- endsemconv -->
 
-[DocumentStatus]: https://github.com/open-telemetry/opentelemetry-specification/tree/v1.22.0/specification/document-status.md
+## Examples
+
+### Chat completion
+
+This example covers the following scenario:
+
+- user requests chat completion from OpenAI GPT-4 model for the following prompt:
+
+  - System message: `You're a friendly bot that answers questions about OpenTelemetry.`
+  - User message: `How to instrument GenAI library with OTel?`
+
+- The model responds with `"Follow GenAI semantic conventions available at opentelemetry.io."` message
+
+Span:
+
+|   Attribute name    |                     Value                             |
+|---------------------|-------------------------------------------------------|
+| Span name           | `"chat.completion gpt-4"`                             |
+| `gen_ai.system`     | `"openai"`                                            |
+| `gen_ai.request.model`| `"gpt-4"`                                           |
+| `gen_ai.request.max_tokens`| `200`                                          |
+| `gen_ai.request.top_p`| `1.0`                                               |
+| `gen_ai.response.id`| `"chatcmpl-9J3uIL87gldCFtiIbyaOvTeYBRA3l"`            |
+| `gen_ai.response.model`| `"gpt-4-0613"`                                     |
+| `gen_ai.usage.completion_tokens`| `47`                                      |
+| `gen_ai.usage.prompt_tokens`| `52`                                          |
+| `gen_ai.response.finish_reason`| `"stop"`                                   |
+
+Events:
+
+1. `gen_ai.system.message`.
+   Not reported if when capturing sensitive content is not enabled.
+
+   |   Property          |                     Value                             |
+   |---------------------|-------------------------------------------------------|
+   | `gen_ai.system`     | `"openai"`                                            |
+   | Event payload       | `{"content":"You're a friendly bot that answers questions about OpenTelemetry."}` |
```

Comment:
A thought I had here is how we might go about giving these messages more structure over time. I recognize that a lot of this really just comes down to the system you're using, as different model APIs will return more or less structure than others, different keys for values, etc. I don't know how we might address this over time but I'd love to get into a world where the experience for most people involves nice, system-readable, structured information.

Reply:
This is an example that shows payload as a json - I did add a bare-minimum structure in the event definitions above and  agree that we need to keep extending it

Reply:
I have the following wording in the event section

> Instrumentations for individual GenAI systems MAY add system specific fields into corresponding events payload. It's RECOMMENDED to document them in system-specific extensions. Telemetry consumers SHOULD expect to receive unknown payload fields.

I can explicitly call out that we expect to add new generic body fields into the payloads, but I feel it's implied and we already warn consumers. WDYT?